make/autoconf/jvm-features.m4

```diff
@@ -1,5 +1,6 @@
 #
 # Copyright (c) 2011, 2022, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2022, IBM Corp.
```

Comment:
I understand the discussion about copyright lines has plagued this PR, but this really looks odd: adding an IBM copyright line when just removing code. That does not make sense to me.

Reply:
+1 Removing code does not add anything copyrightable; I'd remove that copyright line.

Reply:
I agree. I no longer add the IBM copyright for files where a one line change, or only removals took place.